eng/common/TestResources/Remove-TestResources.ps1

```diff
@@ -143,54 +147,40 @@ $exitActions.Invoke()
 <#
 .SYNOPSIS
 Deletes the resource group deployed for a service directory from Azure.
-
```

Comment:
Why was all this whitespace deleted? This makes it less readable and, in some cases, will impact the rendered output.

Reply:
Tracked here: https://github.com/Azure/azure-sdk-tools/issues/622